contracts/minter-suite/Minters/MinterMerkle/MinterMerkleV2.sol

```diff
@@ -122,12 +121,16 @@ contract MinterMerkleV2 is ReentrancyGuard, IFilteredMinterMerkleV0 {
      * which this contract will be a minter.
      * @param _minterFilter Minter filter for which this will be a
      * filtered minter.
+     * @param _delegationRegistryAddress Delegation registry contract address.
      */
-    constructor(address _genArt721Address, address _minterFilter)
-        ReentrancyGuard()
-    {
+    constructor(
+        address _genArt721Address,
+        address _minterFilter,
+        address _delegationRegistryAddress
+    ) ReentrancyGuard() {
         genArt721CoreAddress = _genArt721Address;
         genArtCoreContract = IGenArt721CoreContractV3(_genArt721Address);
+        DELEGATION_REGISTRY_ADDRESS = _delegationRegistryAddress;
```

Comment:
see PR description regarding why events are not emitted here - tldr: if we want to index this off-chain, we should update this to be a MinterMerkleV3

Reply:
After some more thought, totally agree here. I've made the updates and bumped this PR to add a new MinterMerkleV3. This is a more clean solution, and we can make the corresponding subgraph updates when we add MinterHolderV2 to our indexing/frontend.